Quadriga/src/main/webapp/WEB-INF/views/addprojectblogentry.jsp

```diff
@@ -35,33 +13,193 @@
-					</div>
-					<br>
-				</td>
+					</div> <br></td>
 			</tr>
-			
+
 			<tr>
-				<td colspan=2><form:textarea path="title" id="title"
+				<td colspan="2"><form:textarea path="title" id="title"
 						placeholder="Enter Title"
 						style="width: 100%; border : solid 1px; height: 84px; border-color : #D3D3D3; font-weight: bold; font-size: 24px; vertical-align: bottom; align: center; text-align: center; padding : 20px 0" /><br>
 				</td>
 			</tr>
 			<tr>
-				<td colspan=2><form:textarea path="description" id="description" /></td>
+				<td colspan="2"><form:textarea path="description"
+						id="description" /></td>
+			</tr>
+			<tr>
+				<td colspan="1"><button class="btn btn-primary" type="button"
+						data-toggle="collapse" data-target="#networkTable">
+						<i class="fa fa-plus-circle" aria-hidden="true"></i> Add a Network<a></a>
+					</button></td>
 			</tr>
 		</table>
 
+
 		<input type="hidden" name="projectId" value="${project.projectId}">
+		<div id="networkTable" class="collapse">
+			<c:choose>
+				<c:when test="${not empty networks}">
+					<div class="table-responsive">
+						<table class="table table-striped networks">
+							<thead>
+								<tr>
+									<th width="80%">Name</th>
+									<th>Action</th>
+								</tr>
+							</thead>
+
+							<tbody>
+								<c:forEach var="network" items="${networks}">
+									<tr>
+										<td>${network.networkName}</td>
+										<td><a class="btn btn-primary" data-toggle="collapse"
+											href="#networkBox" aria-expanded="false"
+											aria-controls="collapseExample" onclick="loadNetwork()">View
+												Network</a></td>
 
+									</tr>
+								</c:forEach>
+							</tbody>
+						</table>
+						<div id="networkBox" class="collapse" 
+							style="min-height: 500px; width: 100%; text-align: left;">
+							<div id="addImage">
+								<button class="btn btn-primary" type="button">
+									<i class="fa fa-plus-circle" aria-hidden="true"></i> Add This
+									Network To Editor<a></a>
+								</button>
+							</div>
+						</div>
+					</div>
+				</c:when>
+			</c:choose>
+		</div>
 	</form:form>
+
 </article>
 
+
+
+<script
+	src="https://cdn.rawgit.com/cytoscape/cytoscape.js-cose-bilkent/1.0.2/cytoscape-cose-bilkent.js"
+	type="text/javascript"></script>
+<script
+	src="${pageContext.servletContext.contextPath}/resources/js/cytoscape/publicNetwork.js"
+	type="text/javascript"></script>
+<script
+	src="${pageContext.servletContext.contextPath}/resources/js/cytoscape/dist/cytoscape.js"
+	type="text/javascript"></script>
+<script type="text/javascript">
+	function loadNetwork() {
+		$.ajax({
+			type : "GET",
+			contentType : "application/json",
+			datatype : 'text',
+			url : "${pageContext.servletContext.contextPath}/sites/"
+					+ "sampleproj" + "/visualizenetwork/" + 'NETCEnz9o',
+			timeout : 100000,
+			success : function(data) {
+				visualizeNetwork(data);
+			},
+			error : function(e) {
+				console.log(e.Message);
+			}
```

Comment:
the user should see an actual error message if a network couldn't be retrieve for some reason.